### ISSUE TITLE
make selected area more visible

### ISSUE THREAD

**Issue by MCLV-pandorabox** [minetest-mods/areas#61]
It would be so handy if you could actually see the area like the way worldEdit or postool work, and not just position 1 and 2

**Comment by SmallJoker** [comment]
Yes. That would be nice. Feel free to propose a PR if you'd like to work on this feature.

**Comment by fluxionary** [comment]
this is something i'd love myself, but have never gotten around to doing. i'm not sure when i'll get around to it though.

**Comment by tigercoding56** [comment]
worldedit does this.
in the worldedit.mark_region  function of https://github.com/Uberi/Minetest-WorldEdit/blob/master/worldedit_commands/marker.lua
you could just make a optional depends
